sdk/core/amqp-common/package-lock.json

```diff
@@ -0,0 +1,5806 @@
+{
```

Comment:
After rush, we don't need `package-lock.json`. 
Please remove this file.

Reply:
Removed

Reply:
package-lock.json seems still included? Otherwise looks good!

Reply:
Oops, seems to have gotten included since some `npm install` vs `rush' test that we were doing yesterday.. removed in https://github.com/Azure/azure-sdk-for-js/pull/2702